.gitignore

```diff
@@ -130,6 +130,7 @@ $tf/
 _ReSharper*/
 *.[Rr]e[Ss]harper
 *.DotSettings.user
+.idea
```

Comment:
>idea [](http://example.com/codeflow?start=1&length=4)

No objection to the change, but which tool uses this file?

Reply:
JetBrains Rider

Reply:
Cool! How does it compare to VS code or VS?

Reply:
I enjoy it a lot. Nimble and powerful at the same time.